packages/itwin/grouping-mapping-widget/package.json

```diff
@@ -0,0 +1,109 @@
+{
+  "name": "@itwin/grouping-mapping-widget",
+  "version": "0.0.14",
+  "description": "An iTwin.js 3D Viewer Widget that interfaces with the Bentley Reporting Platform.",
```

Comment:
maybe switch this to "iTwin Reporting Platform"?

Reply:
agree